docs/api.md

```diff
@@ -929,6 +929,7 @@ Shortcut for [page.mainFrame().waitForFunction(pageFunction[, options[, ...args]
 - `selector` <[string]> A [selector] of an element to wait for,
 - `options` <[Object]> Optional waiting parameters
   - `visible` <[boolean]> wait for element to be present in DOM and to be visible, i.e. to not have `display: none` or `visibility: hidden` CSS properties. Defaults to `false`.
+  - `hidden` <[boolean]> wait for element to not be found in the DOM or to be hidden, i.e. have `display: none` or `visibility: hidden` CSS properties. Defaults to `false`.
```

Comment:
It seems the same should be added in `frame.waitForSelector(selector[, options])` description.

Reply:
that's part of  PR #967 that's still open. I should have wait to open this one. I make bad decisions late at night, sorry about that